### DISCUSSION TITLE
User presets for brightness/contrast (feature request)

### DISCUSSION THREAD

**Discussion by WWWFWWW** [MonitorControl/MonitorControl#1098] [Ideas]
It would be really great if MonitorControl had custom user presets for specific brightness/contrast values.

I am often working on a calibrated monitor set to a rather low brightness and contrast (set to an approximate industry standard conditions in a darkened room) when editing photographs, with the brightness and contrast determined by hardware calibration for best colour accuracy.

However, when just browsing the web on a sunny day with the window blinds open, I'd like to quickly up the values to some higher value for readability, but still retain my previous fine settings as a preset I could easily return to.

Having two presets (or at least one, "calibrated") would really help with that, especially with keyboard shortcuts for them.

I am aware I could script _ddcctl_ for that, but MonitorControl is so much nicer to use!

**Comment by waydabber** [comment]
I added a feature request based on this: https://github.com/MonitorControl/MonitorControl/issues/1106
